### PR TITLE
WIP:fix(UsersManagement):fix sort by username(bug #902)

### PR DESCRIPTION
**Pull request by doronlnd84**

### CODE REVIEW

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ import UserTypeEnum from 'models/enums/UserType';
 import Language from 'models/Language';
 import StoreStateType from 'redux/storeStateType'
 import axios from 'Utils/axios'
-import { get } from 'Utils/auxiliaryFunctions/auxiliaryFunctions'
-
+import {get} from 'Utils/auxiliaryFunctions/auxiliaryFunctions'
+import {UsersManagementTableHeadersNames } from './UsersManagementTableHeaders';
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:19]: add space

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 2]
```diff
@@ -51,6 +51,14 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
         }
     }
 
+    const fillEmptyfullName= (users: any[]):SignUpUser[]=>{
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:54]: add space

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 3]
```diff
@@ -51,6 +51,14 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
...
+        users.forEach(user=> {
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:55]: add space

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts [hunk 4]
```diff
@@ -51,6 +51,14 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
...
+            if(!Boolean(user.fullName)){
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:56]: Boolean is not needed

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts (hunk 2 above)
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/useUsersManagement.ts:54]: add type

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/UsersManagement/useUsersManagement.ts (hunk 1 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/UsersManagement/useUsersManagement.ts:19]: imports order

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Review by daniel7764 — DISMISSED** [review]

**Review by RoeiRom — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Comment by doronlnd84** [comment]
האיפיון שונה

**Pull request closed without merging**